package.json

```diff
@@ -3,10 +3,11 @@
   "description": "This app provides a backend and frontend for the notification API available in Nextcloud.",
   "version": "3.0.0",
   "author": "Joas Schilling <coding@schilljs.com>",
-  "license": "agpl",
+  "license": "AGPL-3.0-only",
```

Comment:
should be the one from the app so 3.0 or later
It's just a copy paste error from the info.xml code that only allows `apgl`?
https://github.com/nextcloud/appstore/blob/7153c8d0bf021e39548ca519b2efbfe5e86f655b/nextcloudappstore/api/v1/release/info.xsd#L370

Reply:
Since the app still has parts being licensed under `AGPL-3.0-only` the app also needs to be `AGPL-3.0-only`. This can only be changed once all parts of the code are `AGPL-3.0-or-later`.